### PR TITLE
Remove redundancy of 'new_contributor' code

### LINKED ISSUE #3864: add link to "new contributor" labels
Hi, this is a [first-timers-only issue](https://publiclab.github.io/community-toolbox/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** the file [app/models/node_tag.rb](https://github.com/publiclab/plots2/blob/master/app/models/node_tag.rb) in the `plots2` repository (press the little pen Icon) and edit the line as shown below. 

[See this page](https://publiclab.github.io/community-toolbox/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
@@ -21,7 +21,7 @@ def author
 
   def new_author_contributor
     @uid = uid
-    return "<span class = 'label label-success'><i>New Contributor</i></span>".html_safe if Node.where(:uid => @uid).length === 1
+    return "<a href='/tag/first-time-poster' class='label label-success'><i>new contributor</i></a>".html_safe if Node.where(:uid => @uid).length === 1
   end
 
   def user
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### 🤔❓ Questions?

Leave a comment below!

### Is someone else already working on this?

We encourage you to link to this issue by mentioning the issue # in your pull request, so we can see if someone's already started on it. **If someone seem stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://publiclab.github.io/community-toolbox/#r=all). Thanks!

(This issue was created by [First-Timers-Bot](https://github.com/hoodiehq/first-timers-bot).)

### PR DESCRIPTION
**Pull request by meinou**
Fixes #3864  (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by meinou** [comment]
@jywarren
Thanks for being a mentor! 

I found 8 places where `new_author_contributor ` is used. I can go through and replace them all with `new_contributor` or can leave it the way it is.

**Comment by meinou** [comment]
For some reason I missed all deadlines, but I still want to contribute. Would you mind to help me understand why system takes my changes as conflicts? I'm a bit confused. 
Thanks, @jywarren !

**Comment by jywarren** [comment]
Hi, no worries! One thing that can happen is that if other people change the code while you're working, your changes may conflict. But it's pretty easy to resolve. Either you can rebase your work over the latest version:

https://publiclab.org/wiki/contributing-to-public-lab-software#Rewinding+the+master+branch

Or you may be able to click `Resolve conflicts` in the Github interface here. If you get stuck, i'd be happy to help you complete this one!

**Comment by meinou** [comment]
Hi @jywarren , thank you for support! I hope now conflicts are resolved!

File: app/models/node_tag.rb [hunk 1]
```diff
@@ -15,17 +15,16 @@ def increment_count
     tag.save
   end
 
+  def user
+    DrupalUser.find_by(uid: uid).try(:user)
```
Comment (jywarren) [inline app/models/node_tag.rb]: I /think/ this could become `User.find(uid)` to be simpler? Just a tip!

Comment (jywarren) [reply app/models/node_tag.rb]: But this is looking GREAT!

**Comment by jywarren** [comment]
Oh oops! It'll have to be `User.find(uid: ___)` not `id` -- sorry! That should resolve this and then we'll be ready to merge. Thanks!!!

**Comment by jywarren** [comment]
Hi, I added the extra fix i'd gotten wrong but now there is a conflict with @narnt's work in https://github.com/publiclab/plots2/pull/3928#issuecomment-437697299 -- see how I explain in that comment what happened? Would you be able to work together to resolve the conflicts in this PR now, and then we can merge this in? Thank you so much!!!

**Comment by dinaelhanan** [comment]
Hey! Looks like you have a few conflits! Be sure to check that out :) @meinou

**Comment by jywarren** [comment]
Great, i resolved the conflicts and this passed! Thanks a lot!

**Pull request merged**